build.sbt

```diff
@@ -762,7 +773,8 @@ lazy val pkg = (project in file("lib/scala/pkg"))
     version := "0.1",
     libraryDependencies ++= Seq(
       "org.graalvm.truffle" % "truffle-api"      % graalMavenPackagesVersion % "provided",
-      "io.circe"           %% "circe-yaml"       % circeYamlVersion          % "provided",
+      "io.circe"           %% "circe-core"       % circeVersion,
```

Comment:
What do we still use `io.circe:circe-core` for?

Reply:
JSON